Update ghost.mdx with new community resource (#6436)
withastro/docs: src/content/docs/en/guides/cms/ghost.mdx

> - You can get started with this [Astro Ghost Starter](https://github.com/PhilDL/astro-starter-ghost).
> - [Astro + Ghost + Tailwind CSS](https://andr.ec/posts/astro-ghost-blog/) by Andre Carrera
> - [Ghost CMS & Astro](https://matthiesen.xyz/blog/tutorial-ghost-cms-astro) by Adam Matthiesen
> - Try this easy to use Integration and API [@matthiesenxyz/astro-ghostcms](https://github.com/MatthiesenXYZ/astro-ghostcms).

Reviewer: ```suggestion
- [`astro-starter-ghost`](https://github.com/PhilDL/astro-starter-ghost) on GitHub
- [`astro-ghostcms`](https://github.com/MatthiesenXYZ/astro-ghostcms) on GitHub
- [Astro + Ghost + Tailwind CSS](https://andr.ec/posts/astro-ghost-blog/) by Andre Carrera
- [Ghost CMS & Astro Tutorial](https://matthiesen.xyz/blog/tutorial-ghost-cms-astro) by Adam Matthiesen
```
